### ISSUE TITLE
NTR: Thiosulfate high in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8416]
New term request Thiosulfate high in blood (elthat_181102180505): Thiosulphate

**Comment by pnrobinson** [comment]
@MickeySegal
this is found elevated in urine in several diseases but I cannot find the context for blood findings?

**Comment by MickeySegal** [comment]
British spelling:
https://www.ncbi.nlm.nih.gov/books/NBK453432/
"Increased plasma thiosulphate (normal range: <4 μmol/L)"

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000757
- label: Elevated circulating thiosulfate concentration
- parents: [Abnormal blood ion concentration (HP:0003111)](https://hpo.jax.org/app/browse/term/HP:0003111)
- pmid: PMID:28933811
- def: The concentration of thiosulfate in the blood circulation is above the upper limit of normal.
- comment: Thiosulfate is an oxyanion of sulfur with the chemical formula S2O3(2-).
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8416
- synonyms:
